syntax = "proto3";

package greet;

service ProjectService {
	// Unary
	rpc IsUserInProject (ProjectRequest) returns (IsUserInProjectReply);
	rpc SendNotification (CreateNotificationRequest) returns (NotificationReply);
}

message ProjectRequest {
	string project_id = 1;
	string user_id = 2;
}

message IsUserInProjectReply {
   bool result = 1;
}


message CreateNotificationRequest {
	string owner_id = 1;
	int32 type_id = 2;
	string content_id = 3;
	string content = 4;
}


message NotificationReply {
   string id = 1;
}
